src/modules/licence-import/load/licence.js

```diff
@@ -45,15 +45,18 @@ const loadAgreements = async licence => {
 };
 
 const loadPurposeConditions = (purposeId, purpose) => {
-  return Promise.all(purpose.conditions.map(async condition => {
+  Promise.all(purpose.conditions.map(async condition => {
```

Comment:
I feel like we would still want the `return` here, otherwise any errors in the async function will be uncaught.

Also I don't think the `async` is needed above as there is no `await` in this function